WORKSPACE

```diff
@@ -12,10 +12,15 @@ load("@bazel_skylib//lib:unittest.bzl", "register_unittest_toolchains")
 
 register_unittest_toolchains()
 
-load("//tests/import:import_test.bzl", "import_test_repositories")
+# Dependencies of the tests module.
+local_repository(
+    name = "rules_sh_tests",
+    path = "tests",
+)
```

Comment:
Interestingly, this prevents Bazel from traversing `tests` as a package of `@rules_sh` itself in workspace mode. However, in bzlmod that is not the case, as evidenced by the fact that the following command will list targets under `//tests/...`:
```
bazel query //... --experimental_enable_bzlmod
```
You can avoid that by adding a top-level file `.bazelignore` that contains `tests`.

Reply:
With the tests run from `tests/` this import and the `import_test_repositories` can be removed. There is no reason to duplicate those deps when they're only needed under `rules_sh_tests`.

Reply:
Yep removed that part.